sources/platform/integrations/webhooks/ad_hoc_webhooks.md

```diff
@@ -50,9 +56,30 @@ await Actor.addWebhook({
 await Actor.exit();
 ```
 
-To learn more, see the [Apify SDK documentation](/sdk/js/api/apify/class/Actor#addWebhook).
+</TabItem>
+<TabItem value="Python" label="Python">
+
+```python
+from apify import Actor
+
+async def main():
+    async with Actor:
+        other_dataset = await Actor.add_webhook(
+            event_types=['ACTOR.RUN.FAILED'],
+            request_url='https://example.com/run-failed',
+        )
+        # ...
+```
+
+</TabItem>
+</Tabs>
+
+To learn more, see the [JavaScript SDK documentation](/sdk/js/api/apify/class/Actor#addWebhook) or the [Python SDK documentation](/sdk/python/api/apify/class/Actor#add_webhook).
 
-To ensure that duplicate ad-hoc webhooks won't get created in a case of actor restart you can use the `idempotencyKey` parameter. The idempotency key must be unique across all the webhooks of a user so that only one webhook gets created for a given value. You can use, for example, the actor run ID as idempotency key:
+To ensure that duplicate ad-hoc webhooks won't get created in a case of Actor restart you can use the idempotency key parameter. The idempotency key must be unique across all the webhooks of a user so that only one webhook gets created for a given value. You can use, for example, the Actor run ID as idempotency key:
```

Comment:
```suggestion
To ensure that duplicate ad-hoc webhooks won't get created in the case of Actor restart, you can use the idempotency key parameter. The idempotency key must be unique across all the webhooks of a user so that only one webhook gets created for a given value. You can use, for example, the Actor run ID as an idempotency key:
```